Author: Sergey Slipchenko <faergeek@gmail.com>
Description:
 "Run" dialog always in background if started with Alt+F2 (Closes: #656672).

diff -Naurp lxpanel.orig/src/gtk-run.c lxpanel/src/gtk-run.c
--- lxpanel.orig/src/gtk-run.c	2011-07-29 14:37:01.000000000 +0200
+++ lxpanel/src/gtk-run.c	2012-04-20 14:38:44.883785353 +0200
@@ -344,7 +344,7 @@ void gtk_run()
     gtk_widget_show_all( win );
 
     setup_auto_complete( (GtkEntry*)entry );
-    gtk_widget_show( win );
+    gtk_window_present(GTK_WINDOW(win));
 
     g_signal_connect(entry ,"changed", G_CALLBACK(on_entry_changed), img);
 
